src/app/tests/suites/certification/Test_TC_WNCV_3_2.yaml

```diff
@@ -176,22 +174,14 @@ tests:
               minValue: 10
```

Comment:
hasMasksSet: should be used here instead of ranges

Reply:
Updated please verify it once.